samples/bluetooth/direction_finding_connectionless_tx/CMakeLists.txt

```
@@ -0,0 +1,14 @@
+#
+# Copyright (c) 2021 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+
+cmake_minimum_required(VERSION 3.13.1)
+
+find_package(Zephyr REQUIRED HINTS $ENV{ZEPHYR_BASE})
+
+project(df_beacon)
```

Maybe update this name now:
```suggestion
project(direction_finding_connectionless)
```

---

corrected